src/requirements/Sound/SoundRequirement.tsx

```diff
@@ -0,0 +1,157 @@
+import { Img } from "@chakra-ui/react"
+import Link from "components/common/Link"
+import { RequirementComponentProps } from "requirements"
+import Requirement from "requirements/common/Requirement"
+import slugify from "slugify"
+import useSWRImmutable from "swr/immutable"
+
+const SoundRequirement = ({ requirement, ...rest }: RequirementComponentProps) => {
+  const { data: artistsData, isValidating: artistsLoading } = useSWRImmutable(
+    `/api/sound-artists?searchQuery=${requirement.data?.id}`
+  )
+
+  const { data: songsData, isValidating: songsLoading } = useSWRImmutable(
+    `/api/sound-songs?id=${artistsData?.map((artist) => artist[0].id)}`
+  )
+
+  const songImageUrl = songsData
+    ?.filter((title) => title[0].title == requirement.data.title)
```

Comment:
what's title[0].title? It's bad naming. But the endpoint shouldn't return an unnecessary multidimensional array in the first place